### DISCUSSION TITLE
What was the finishing order?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#401] [Galaxy Trivia] [answered]
Five people were eating 🍎🍏, `A` finished before `B`, but behind `C`. `D` finished before `E`, but behind `B`.

**Comment by peterbaccaro** [comment]
C, A , B, D, then in last E ate apples.

**Comment by thuongtruong1009** [comment, marked as answer]
Based on the information given, the finishing order of the five people eating apples is:

C
A
B
D
E
Here's how we can arrive at this answer:

A finished before B, but behind C: This tells us that C finished first, followed by A, and then B.
D finished before E, but behind B: This tells us that B finished before D, and then D finished before E.
Combining these two pieces of information, we can put together the complete finishing order as shown above.

**Comment by alirezazahiri** [comment]
> Five people were eating 🍎🍏, `A` finished before `B`, but behind `C`. `D` finished before `E`, but behind `B`.

C A B D E

**Reply by ParanoidUser** [reply]
Bingo! One more to go for the `Galaxy Brain` badge. 😊

**Comment by alirezazahiri** [comment]
hey ParanoidUser/yolo, thanks for your help
